avm/res/web/site/basic-publishing-credentials-policy/main.json

```diff
@@ -0,0 +1,87 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "metadata": {
+    "_generator": {
+      "name": "bicep",
+      "version": "0.23.1.45101",
```

Comment:
Be aware - the current latest version is `0.24.24`

Reply:
Thanks., updated